charts/iter8/templates/_task-slack.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "task.slack" -}}
+{{- /* Validate values */ -}}
+{{- if not . }}
+{{- fail "slack notify values object is nil" }}
+{{- end }}
+{{- if not .url }}
+{{- fail "please set a value for the url parameter" }}
+{{- end }}
+# task: send a Slack notification
+- task: notify
+  with:
+    url: {{ .url }}
+    method: POST
+    payloadTemplateURL: {{ default "https://github.com/iter8-tools/iter8/master/blob/slack-template.tpl" .payloadTemplateURL }}
```

Comment:
Is this where the "default" slack template will be located for v0.11? I don't see it there. Perhaps it should be located under .../iter8/master/blob/templates/slack-template.tpl (and the tpl files should also be added)?

Reply:
Actually, it makes sense to locate them here: https://github.com/iter8-tools/iter8/tree/master/charts/iter8/templates

Reply:
Fixed

Reply:
@sriumcp Should this be tied down to a particular commit/branch? Because once we move to v0.12, this will be gone right? And this reference will not work anymore.

Reply:
You are correct. We want to tie it to a commit. So, perhaps, let us push the templates first, create a commit, and then update/merge this?

Reply:
Okay sounds good